package.json

```diff
@@ -208,7 +209,8 @@
 		"test": "npm run lint && npm run test-unit",
 		"pretest-e2e": "npm run env reinstall",
 		"test-e2e": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
-		"test-e2e:watch": "npm run test-e2e -- --watch",
+		"test-e2e:local": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
```

Comment:
I don't see any difference between `test-e2e` and `test-e2e:local`? Do I miss something?

Reply:
something important (just kidding). The difference is that if you run `local` it won't run the `pretest` which is annoying if you have a custom env setup. I think `npm test-e2e` could just do `npm run test-e2e:local` to avoid duplication though.

Reply:
You still need to reset the database because you will see test failures caused by the tests which add taxonomies, categories, etc. So it's mostly the issue with how `pretest-e2e` works at the moment.

Reply:
no you don't, I run tests everyday and I never use `pretest` and everything works properly.

Reply:
Do you run all tests or just the one file which is related to your work? 😄

Reply:
it depends I can do both :)

Reply:
>The difference is that if you run local it won't run the pretest which is annoying if you have a custom env setup.

That was the goal of this change, yes.